Automattic/wp-calypso
# Quick Switch: Remove modal announcement

#### Changes proposed in this Pull Request

* Reverts https://github.com/Automattic/wp-calypso/pull/54687
* Some of the spec files no longer exist.

#### Testing instructions



* Change your `nav-unification-quick-switch-modal` preference to `0` via the staging badge in the bottom right.
* Before applying these changes, check that the quick switch modal appears on a page that can switch, like the Posts page:

<img width="833" alt="Screen Shot 2021-07-22 at 15 03 00" src="https://user-images.githubusercontent.com/1233880/126643568-526bc176-df9c-4a99-8b46-43518c3abc7a.png">

* After applying these changes, check that the modal does not appear.



Fixes https://github.com/Automattic/wp-calypso/issues/55690
